backend/Perflow/DataAccess/Context/ModelBuilderExtensions.cs

```diff
@@ -20,6 +25,361 @@ public static void Configure(this ModelBuilder modelBuilder)
+            new List<PlaylistSong>
+            {
+                new PlaylistSong
+                {
+                    Id=1,
+                    PlaylistId = playlists[0].Id,
+                    SongId = songs[0].Id,
+                    CreatedAt =new DateTimeOffset(new DateTime(2020, 10, 19))
+                },
+                new PlaylistSong
+                {
+                    Id=2,
+                    PlaylistId = playlists[0].Id,
+                    SongId = songs[1].Id,
+                    CreatedAt =new DateTimeOffset(new DateTime(2020, 10, 19))
+                },
+                new PlaylistSong
+                {
+                    Id=3,
+                    PlaylistId = playlists[0].Id,
+                    SongId = songs[2].Id,
+                    CreatedAt =new DateTimeOffset(new DateTime(2020, 10, 19))
+                },
+                new PlaylistSong
+                {
+                    Id=4,
+                    PlaylistId = playlists[1].Id,
+                    SongId = songs[0].Id,
+                    CreatedAt=new DateTimeOffset(new DateTime(2020, 2, 1))
+                },
+                new PlaylistSong
+                {
+                    Id=5,
+                    PlaylistId = playlists[1].Id,
+                    SongId = songs[1].Id,
+                    CreatedAt=new DateTimeOffset(new DateTime(2020, 2, 1))
+                },
+                new PlaylistSong
+                {
+                    Id=6,
+                    PlaylistId = playlists[1].Id,
+                    SongId = songs[2].Id,
+                    CreatedAt=new DateTimeOffset(new DateTime(2020, 2, 1))
+                },
+                new PlaylistSong
+                {
+                    Id=7,
+                    PlaylistId = playlists[2].Id,
+                    SongId = songs[1].Id,
+                    CreatedAt=new DateTimeOffset(new DateTime(2020, 12, 1))
+                },
+                new PlaylistSong
+                {
+                    Id=8,
+                    PlaylistId = playlists[2].Id,
+                    SongId = songs[2].Id,
+                    CreatedAt=new DateTimeOffset(new DateTime(2020, 12, 1))
+                }
+                ,
+                new PlaylistSong
+                {
+                    Id=9,
+                    PlaylistId = playlists[2].Id,
+                    SongId = songs[3].Id,
+                    CreatedAt=new DateTimeOffset(new DateTime(2020, 12, 1))
+                }
+            };
+
+        private static string HashPassword(string password, byte[] salt) =>
+            Convert.ToBase64String(
+                    KeyDerivation.Pbkdf2(
+                        password: password,
+                        salt: salt,
+                        prf: KeyDerivationPrf.HMACSHA256,
+                        iterationCount: 10000,
+                        numBytesRequested: 256 / 8
+                        )
+                );

```

Comment:
Use const for magic values (256/8 = 32, 10000).
And this would be better to separate this method from class and move it to helper

Reply:
![image](https://user-images.githubusercontent.com/24195200/127612689-b1c8ffec-c860-4168-9a5e-8c53420d91f9.png)
Also, pay attention on parentheses

Reply:
helper created